vector-im/element-android
# Start DM on first message (UI)

## Type of change


## Content

This PR displays a "fake" local timeline after selecting the users to create a DM. Sending a message or event is not handled yet, so the real room cannot be created for now.
The flow is hidden behind a debug feature flag accessible from the debug menu.

## Motivation and context

UI: https://github.com/vector-im/element-android/issues/5525

## Screenshots / GIFs

https://user-images.githubusercontent.com/11990514/177182407-61af8ca8-e78e-4bf7-b01b-3bbe8aef492f.mp4

## Tests

- Activate the start DM feature in the debug menu -> Features
- On the home screen, select the first tab (Direct Messages)
- Click on the fab button to start a DM, select several users and press "GO"
- A kind of timeline should now appear, there is no action in the toolbar (except the close button)
- It is not possible to access the room settings or details
- Composer actions are not working (an error appears when we try to send an event to the room)
- When we go back to the room list or relaunch the app, the "fake" room is deleted from the DB and disappears from the room list

## Tested devices

- OS version(s): API 32

## Checklist
